src/System.Web.Http.Cors/CorsHttpConfigurationExtensions.cs

```diff
@@ -26,16 +26,38 @@ public static class CorsHttpConfigurationExtensions
         /// <param name="httpConfiguration">The <see cref="HttpConfiguration"/>.</param>
         public static void EnableCors(this HttpConfiguration httpConfiguration)
         {
-            EnableCors(httpConfiguration, null);
+            EnableCors(httpConfiguration, null, false);
```

Comment:
FYI we've got a meeting scheduled next Tuesday to decide whether default should be `true` i.e. whether we should break compatibility to ensure `IExceptionHandler` sees what it should. Will let you know&hellip;

Reply:
Thanks.

Reply:
Decision was to avoid breaking changes and stick with the approach you've implemented here.